src/misc/8.md

```diff
@@ -0,0 +1,14 @@
+# Misc 8 @Darksonn
+
```

Comment:
maybe worth adding an explicit question about whether it compiles or not

Reply:
realistically all the MD files should have equivalent notes to the unsafe ones